jth-tests/pom.xml

```diff
@@ -88,6 +115,20 @@
                     <skip>true</skip>
                 </configuration>
             </plugin>
+            <!-- Since this is not an hpi packaged module, it needs to be told explicitly to resolve the .hpi of depending plugins for ExternalJenkinsRule -->
+            <plugin>
+                <groupId>org.jenkins-ci.tools</groupId>
+                <artifactId>maven-hpi-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>default-resolve-test-dependencies</id>
+                        <phase>test-compile</phase>
+                        <goals>
+                            <goal>resolve-test-dependencies</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
```

Comment:
`ExternalJenkinsRule` depend on maven pulling all those .hpis which happen for normal plugin module tests but since this is dedicated module, it needs to be invoked specifically.